specification.md

```diff
@@ -3539,10 +3539,10 @@ If `false`, both Event payload and context attributes should be accessible.
 
 ##### Auth Definition
 
-Auth definitions can be used to define authentication information that should be applied
+Auth definitions can be used to define authentication information that is applied
 to resources defined in the operation property of [function definitions](#Function-Definition).
-It is not used as authentication information for the function invocation, but just to access
-the resource containing the function invocation information.
+
+If defined, it will replace the [security scheme](https://spec.openapis.org/oas/v3.1.0#security-scheme-object) described in the OpenAPI file.
```

Comment:
What do we mean by replacing the security scheme? The OpenAPI file describes the contract of a particular service, how can we replace this? Do you mean that instead of using the security scheme defined by the OpenAPI to call the service it would use the one defined in this section?

Reply:
That's the original desired behavior, yes. But based on [this conversation](https://github.com/serverlessworkflow/specification/issues/612#issuecomment-1085051096), we are likely to restructure the way about auth definitions, so you could define auth info for fetching the spec resource and another to replace the original security scheme in the spec if desired.

Reply:
Okay, but what do you mean by the `original security scheme`? The one in the OpenAPI? Or the one defined to fetch the spec? I think there is some confusion here over what is overriding what if someone reads the spec without the context of that issue.

Reply:
The on in the Open API. This text is linked to the Open API spec doc. This will be enlightened as soon as we add the property to separate the resource vs invocation auth to the spec.